src/clib/lib/config/config_parser.cpp

```diff
@@ -657,15 +658,33 @@ config_parse(config_parser_type *config, const char *filename,
         hash_iter_free(keys);
     }
 
-    if (util_file_readable(filename)) {
+    std::ifstream file_handler;
+    file_handler.exceptions(std::ifstream::failbit | std::ifstream::badbit);
+    bool file_readable_check_succeeded = true;
+    try {
+        file_handler.open(filename);
+    } catch (std::ios_base::failure &err) {
+        file_readable_check_succeeded = false;
+        auto error_message = fmt::format(
+            "could not open file `{}` for parsing - {}", filename, err.what());
```

Comment:
This failure condition is untested now right?

Reply:
AHHHH thank you eivind. guess what. trying to add a test i've discovered, why, yes, you guessed it, yet another bug :exploding_head:

while working on these changes, i did the dumbs and got a bunch of tests failing, one of them something with observations. that's how i figured that ah, we use the parser for parsing the observation config (i had done the dumbs in the confuig parser, you see)

so i figured i make a test where the obs config is not readable. UTIL ABORT! :tada:

Reply:
hm the OBS_CONFIG file seems to (at least at first, not sure about what happens after the util abort) parsed by `ecl`'s `lib/util/parser.cpp` - so maybe i just need to make an issue about this, and try to trigger / cover the config parsing error another way